### ISSUE TITLE
Diktat rule set should not trigger standard ruleset when is called by ktlint

### ISSUE THREAD

**Issue by akuleshov7** [cqfn/diKTat#69]

**Comment by petertrr** [comment]
ktlint with `--debug` shows:
```
ktlint:
   [ktlint] [DEBUG] Discovered ruleset with " standard" id.
   [ktlint] [DEBUG] Initializing "plain" reporter with {verbose=false, color=false, color_name=DARK_GRAY}
   [ktlint] [DEBUG] Discovered ruleset with "diktat-ruleset" id.
```

**Comment by petertrr** [comment]
Maven configuration has been fixed in #90 by excluding `ktlint-ruleset-standard` from dependencies.

**Comment by akuleshov7** [comment]
closed by configuration by @petertrr
